ui/src/containers/CreateVolume.js

```diff
@@ -568,14 +570,25 @@ const CreateVolume = (props) => {
                             onChange={(e) => {
                               setLabelValue(e.target.value);
                             }}
+                            onBlur={handleOnBlur}
                           />
-                          <Button
-                            text={intl.translate('add')}
-                            type="button"
-                            onClick={addLabel}
-                            data-cy="add-volume-labels-button"
-                            outlined
-                          />
+                          {touched.labelName && touched.labelValue ? (
+                            <Button
+                              text={intl.translate('add')}
+                              type="button"
+                              onClick={addLabel}
+                              data-cy="add-volume-labels-button"
+                              variant={'buttonSecondary'}
+                            />
+                          ) : (
+                            <Button
+                              text={intl.translate('add')}
+                              type="button"
+                              onClick={addLabel}
+                              data-cy="add-volume-labels-button"
+                              outlined
+                            />
```

Comment:
shouldn't we disable the button initially instead of just changing it's color ? Also I think we can remove the onClick handler in this case

Reply:
Done in https://github.com/scality/metalk8s/pull/3338/commits/7a961e20d30abe2e3ea76d4d7050fbe7bc9464c4 ;)